pyproject.toml

```diff
@@ -47,6 +29,36 @@ Source = "https://github.com/IntelLabs/MART"
 mart = "mart.__main__:main"
 
 [project.optional-dependencies]
+
+# These are required dependencies, but we make it flexible for users to adjust.
+core = [
+  "torch >= 2.0.1",
+  # TODO: Move torchvision to the optional vision group, and make optional tests locally, but required tests on CI.
+  "torchvision >= 0.15.2",
+  # TODO: Move pycocotools to the optional objdet group, and make optional tests.
+  "pycocotools ~= 2.0.5",      # data format for object detection.
```

Comment:
why does this not live in `objdet` below?

Reply:
I plan to make the change in a separate PR, because we also need to change the code to make `mart` importable without the `objdet` dependencies.